src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -1957,6 +1957,10 @@ void MessageHelper::SendSystemRequestNotification(
 
   content[strings::params][strings::connection_key] = connection_key;
 
+  policy::PolicyHandlerInterface& policy_handler = app_mngr.GetPolicyHandler();
+  content[strings::msg_params][strings::timeout] =
+      policy_handler.TimeoutExchangeSec();
+
```

Comment:
@jacobkeeler thank you for this note. I grab correct changes from this pull request and revert my provided. Fixed in https://github.com/smartdevicelink/sdl_core/pull/1607/commits/638d17aa4821c5d1eb4b4e9009c027086d26d88a